kuma/attachments/tests/test_views.py

```diff
@@ -273,4 +273,3 @@ def test_raw_file_if_modified_since(client, settings, file_attachment):
     assert 'Cache-Control' in response
     assert 'public' in response['Cache-Control']
     assert 'max-age=900' in response['Cache-Control']
-    assert 'Vary' not in response
```

Comment:
I found that the `Vary` header was now in the response with a value of `Accept-Encoding`. I'm not sure if it's worth checking any of the headers when the response is a 304.